twingate/README.md

```diff
@@ -0,0 +1,95 @@
+# Twingate
+
+## Overview
+[Twingate][1] is a zero trust network access platform that allows fast growing companies to quickly and easily provide secure access to their AWS environment. Incorporating modern technologies like NAT traversal, QUIC, private proxies, and split tunneling, Twingate can replace a traditional or cloud VPN while improving user performance and overall security.
+
+This integration allows organisation to monitor users' resource access activities in real-time.
+
+## Setup
+### Pre-requisite
+1. Twingate Account
+2. Twingate Connector deployed
+
+### Enable Twingate Connector Analytic Logs
+Follow the steps to enable [Real-time Connection Logs][3]
+
+### Install Datadog Agent
+Follow the steps to install [Datadog Agent][4] on the Twingate Connector server
+
+### Configure Datadog Agent
+#### Systemd
+1. Set up [Datadog journald integration][5]
+2. Replace `journald.d/conf.yaml` with the configuration below:
+    ````
+    logs:
+      - type: journald
+        container_mode: true
+        include_units:
+          - twingate-connector.service
+        log_processing_rules:
+        - type: include_at_match
+          name: analytics
+          pattern: ANALYTICS
+            - type: mask_sequences
+          name: remove_analytics
+          replace_placeholder: ""
+          pattern: "ANALYTICS "
+    ````
+3. Add the `dd-agent` user to the `systemd-journal` group `usermod -a -G systemd-journal dd-agent`
```

Comment:
If this is the command to run: `usermod -a -G systemd-journal dd-agent` 

It would be good to specify. Ex:  Add the `dd-agent` user to the `systemd-journal` group using the command `usermod -a -G systemd-journal dd-agent`